types.go

```diff
@@ -328,6 +330,11 @@ type SearchResponse struct {
 	HitsPerPage        int64         `json:"hitsPerPage,omitempty"`
 	Page               int64         `json:"page,omitempty"`
 	TotalPages         int64         `json:"totalPages,omitempty"`
+	IndexUid           string        `json:"indexUid,omitempty"`
```

Comment:
This is not the case when performing a simple `search`. `MultiSearchResponse` should be a union of `SearchResponse` and

```
	IndexUid           string        `json:"indexUid,omitempty"`
```

Reply:
That is why I used `omitempty` which meant that go will not create the fields if it doesn't exist.
What do you mean by Union? The type `Union` in golang or something like:
```
type MultiSearchResponse struct {
	Results []MultiSearchUnionResponse `json:"results"`
}

type MultiSearchUnionResponse struct {
	SearchResponse SearchResponse{}
	IndexUid string
}
```
?
Because if I use the second option, you will have to do something like:
```
myResponse.Results[0].SearchResponse.Limit
```
Instead of:
```
myResponse.Results[0].Limit
```
Like it is in the response API